spec/internal_spec.rb

```diff
@@ -65,10 +74,18 @@
         it { is_expected.to include('<img src="http://example.com', "blobs") }
       end
 
-      context "when record is previewable" do
-        let(:record) { user_with_previewable_image.avatar.preview(resize_to_limit: [500, 500]) }
```

Comment:
We probably should add another example with a mismatched content type (e.g. `text/plain`) to cover the case of falsey `content_type` check in `#applicable_variation?`

Reply:
It will just throw [ActiveStorage::InvariableError](https://edgeapi.rubyonrails.org/classes/ActiveStorage/InvariableError.html)

Reply:
True, unless someone will extend `ActiveStorage.variable_content_types` like we suggest for imgproxy pro with some different content types, seems like a pretty rare case though

Reply:
We can't prevent people from shooting themselves in their foots 🙂